test/sde/sde_twodimlinear_tests.jl

```diff
@@ -103,10 +107,13 @@ sim2 = test_convergence(dts,prob,WangLi3SMil_F(),trajectories=100)
 
 print(".")
 
+Random.seed!(100)
 eigen_est = (integrator) -> integrator.eigen_est = 10.0
 for Alg in [SROCK1, SROCK2], alg in [Alg(), Alg(eigen_est=eigen_est)]
-  sim2 = test_convergence(dts,prob,alg,trajectories=100)
+  local sim2
+  sim2 = test_convergence(dts,prob,alg,trajectories=150)
   @test abs(sim2.𝒪est[:l∞]-1) < 0.2
+  @show sim2.𝒪est[:l∞]
```

Comment:
It's failing here (0.22 instead of <0.2  with the old version using `trajectories =100` and no re-seeding). Locally, the tests are actually passing on my machine.

Reply:
ehh seems like just random variation

Reply:
150 trajectories is still pretty low